src/components/SelectViewOption/SelectViewOption.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./SelectViewOption.css";
+import { ViewOption } from "../../types/types-index";
```

Comment:
when you're importing type, should specify type here.
```suggestion
import type { ViewOption } from "../../types/types-index";
```